var/spack/repos/builtin/packages/py-simpy/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PySimpy(PythonPackage):
+    """SimPy is a process-based discrete-event simulation framework based on standard Python."""
+
+    homepage = "https://simpy.readthedocs.io/"
+    pypi = "simpy/simpy-4.0.2.tar.gz"
+
+    version("4.1.1", sha256="6d8adc0229df6b02fb7e26dcd1338703b4f4f63f167a5ac2a7213cb80aba4484")
```

Comment:
```suggestion
    version("4.1.1", sha256="06d0750a7884b11e0e8e20ce0bc7c6d4ed5f1743d456695340d13fdff95001a6")
```

Reply:
Where did you get that hash? I found `6d8adc0229...` from https://pypi.org/project/simpy/#copy-hash-modal-bf5289a4-c487-45af-a0c3-fb11fa764679

Reply:
![image](https://github.com/spack/spack/assets/4656391/4271d360-68a8-4536-928d-d2c22b675648)

Reply:
But also `spack checksum py-simpy 4.1.1` should return the version line to use. You had the exact same hash for 4.1.1 and 4.0.2, so that was suspicious.

Reply:
Oh! I can't read! Yeah, you are correct, the hash I have is wrong. Will fix.